README.md

```diff
@@ -1354,7 +1469,7 @@ Like the web component differences mentioned above, Skate only polyfills and val
 
 
 
-## Polymer Differences
+## VS Polymer
```

Comment:
Needs revising. Some points to use:

- Polymer has a fundamentally different programming model (functional etc)
- Performance, referencing that Polymer uses the webcomponentsjs polyfills and ours are faster
- Polyfills are outdated (uses v0 API, Skate is up to date)
- Polymer isn't on NPM, only Bower
- Polymer uses HTML imports for their codebase (list downsides as well as it's probably not going to be a spec anymore)

More?